ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml

```diff
@@ -195,10 +195,46 @@ tests:
         and not @fips and @destructive'
       OCP_ARCH: arm64
       TAG_VERSION: '@4.12'
-      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial&;Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-with-ep-sts-ec2-elb
+- as: aws-ipi-disconnected-cco-manual-sts-p2-f14
+  cron: 33 13 14,29 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips'
+      OCP_ARCH: arm64
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service
+- as: aws-ipi-disconnected-cco-manual-sts-p3-f28-destructive
+  cron: 7 1 12 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips and @destructive'
+      OCP_ARCH: arm64
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&
```

Comment:
Shall we append `~MicroShiftOnly&` here?

Reply:
`MicroShiftOnly` was added by new PR #35244, I will review and update my jobs.

Reply:
@kuiwang02 what kind of jobs should add `~MicroShiftOnly&`, per #35244 , I saw it's not required for all jobs.

Reply:
@yunjiang29 there is new kind of cases into golang for microshift production. here is the detailed: https://redhat-internal.slack.com/archives/CPJQCSN49/p1673244238348199

currently all tests defined  are not for microshift production, but the golang case code will come soon.
We need to exclude it for all current tests. I already update golang step to execlude it with PR https://github.com/openshift/release/pull/35243
So, 
1, . if the test take default filter value (not define TEST_FILTERS in test), it will exclude such case by default.
2, for some tests, it can not take default value, and define TEST_FILTERS, so need update to add more one filter `~MicroShiftOnly&`

currently microshift case run only special jenkins pipeline. I was told they will added into prow ci late.
if it is done in prow ci, it will not take default value, and should define TEST_FILTERS for such micorshift test, mayb like `TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;MicroShift&`
note: it is not `~MicroShiftOnly&` and it is `MicroShift` to select microshift case.

hope the above is clear. if not, please free to ping me.

Reply:
@kuiwang02 it's very clear, good to know this new change, thanks!

Reply:
@jianlinliu added `~MicroShiftOnly&` to the job config, PTAL, thanks